tasks/test.py

```diff
@@ -1296,6 +1296,9 @@ def parse_test_log(log_file):
             if json_line["Action"] == "pass" and "Test" in json_line:
                 n_test_executed += 1
                 if f'{json_line["Package"]}/{json_line["Test"]}' in failed_tests:
-                    failed_tests.remove(f'{json_line["Package"]}/{json_line["Test"]}')
-
+                    failed_tests = [
+                        failed_test
+                        for failed_test in failed_tests
+                        if f'{json_line["Package"]}/{json_line["Test"]}' != failed_test
+                    ]
```

Comment:
❓ Question :
More by curiosity, but why `remove` caused flake ? The only difference between what you added and remove is that remove removes the first occurence and here it remove all occurence. But is it possible to have multiple entry of the same `Package/Test` ?

Reply:
Yes the test can be retried more than once. WIth the current implementation if a test is retried three times, fails twice and then succeed, remove will only remove one occurences and the test will be in the failed tests list while it succeeded at the end. I modified the code to rather not have duplicates in the `failed_tests` list